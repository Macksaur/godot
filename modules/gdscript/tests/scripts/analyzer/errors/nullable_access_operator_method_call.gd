class A:
    var b: A?
    func banana():
        pass
func test():
    var foo := A.new()
    foo?.b.banana()
