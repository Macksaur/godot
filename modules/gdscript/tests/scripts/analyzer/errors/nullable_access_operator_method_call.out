GDTEST_ANALYZER_ERROR
Potentially unsafe for nullable "b" to call "banana()".
